MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -68,8 +73,9 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
 Note: Frontend criteria apply to both modules and shared libraries. 
 
 * If provided, End-to-end tests must be written in an [officially supported technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)[^1] (3, 4)
-  * -_note: while it's strongly recommended that modules implement integration tests, it's not a requirement_
+  * -_note: If the new module works together with existing modules, the team must provide a result of an integration test.
```

Comment:
Why has this been changed within the scope of the review of static code analysis?

What is the motivation for making this mandatory if a module has any interaction with another module?

Reply:
Yes. Julian and I think that it is fair to write integration tests (mandatory) if the module is not stand-alone. If it is not a requirement, it does not even need to be mentioned in the acceptance criteria, we think.

Reply:
> My definition of that term in this context is a automated test that demonstrates multiple modules interacting with each other

Yes, that fits to what I have in mind.